src/pages/Forms.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import routes from 'constants/routesPaths';
+import {
+  getFormCategories,
+  getFormsByCategory,
+  getFormError,
+  getFormIsFetching
+} from 'selectors/formSelectors';
+import { fetchFormCategories, fetchForms } from 'actions/formActions';
+import Header from 'components/common/Header';
+import Title from 'components/common/Title';
+import Subtitle from 'components/common/Subtitle';
+import Link from 'components/common/Link';
+import Loading from 'components/common/Loading';
```

Comment:
Maybe you can export all of these from `common`.

Reply:
Sounds great, gonna do it